### ISSUE TITLE
Question: the created graph have been added an origin(0,0,0) by accident? 

### ISSUE THREAD

**Issue by Robinxbnie** [networkx/networkx#3712]
I just used the nx to build a graph and display it in paraview like this:
![image](https://user-images.githubusercontent.com/25780361/68448210-fc260780-019f-11ea-9804-eda7cee9333e.png)

This looks good and shows my nodes as expected. While when show the edges, it looks like below:
![image](https://user-images.githubusercontent.com/25780361/68448260-24156b00-01a0-11ea-8b87-33cb79a9c712.png)

The problem is: it seems nx added an extra edge( from (0,0,0) to one of my node) the down left corner in the picture. This case is neglectable. 
While I have another model which is like below:
![image](https://user-images.githubusercontent.com/25780361/68448345-722a6e80-01a0-11ea-8b23-c1d93da9bf87.png)

In this model, when I show the connectivity/edges, it will look like below:
![image](https://user-images.githubusercontent.com/25780361/68448378-8ec6a680-01a0-11ea-92f3-f78796cf675b.png)

In this case, the exta edges are crazy, so many that we can not neglect them.

My question, what mistake I may have made. We do not have the node at point (0,0,0). How to get ride of this "virtual" node, so get ride of the fake connectivity?

**Comment by dschult** [comment]
That looks like an artifact of your drawing program, not of NetworkX. You can check by seeing if the  NetworkX graph has any nodes with position (0,0,0). 

This is just a guess, but perhaps your drawing program starts lines at the origin by default unless you specify that the first line should start somewhere else. I have not used paraview.

**Comment by Robinxbnie** [comment]
Thanks Dan for your feedback.
I have checked the graph specifically try to delete the node at (0,0,0), confirmed that there is no such a node in the graph.
Yes, so it is mostly likely related to the drawing... I am studying on the drawing, and find the drawing pragram used the networkx class "<class 'networkx.classes.reportviews.EdgeView'>". Still working on it and try to figureout what pit I just falled.
